src/main/java/com/powsybl/openloadflow/network/impl/LfLegBranch.java

```diff
@@ -112,6 +113,16 @@ public boolean hasPhaseControlCapability() {
         return Collections.emptyList();
     }
 
+    @Override
+    public List<LfLimit> getLimits1(final LimitType type) {
```

Comment:
You are not able to get active power and apparent power limits on network side of a three windings transformer?

Reply:
Same as DanglingLine, I will match the implementation to the one of LfBranchImpl.